python/ql/src/semmle/python/web/stdlib/Request.qll

```diff
@@ -0,0 +1,126 @@
+/**
+ * Provides the sources and taint-flow for HTTP servers defined with standard library (stdlib).
+ * Specifically, we model `HttpRequestTaintSource`s from instances of `BaseHTTPRequestHandler`
+ * (or subclasses) and form parsing with `cgi.FieldStorage`.
```

Comment:
I keep reading "form" as a verb in this sentence ("We ... form parsing with ..."). Perhaps change `with` to `using`?

Reply:
Done